strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -223,11 +225,11 @@ def thermal_loss(self, T, nbar, mode):
         self.circuit.thermal_loss(T, nbar, mode)
 
     def measure_fock(self, modes, shots=1, select=None):
+        if select is not None:
+            raise NotImplementedError(
+                "Gaussian backend currently does not support " "postselection"
+            )
```

Comment:
Can't we support post-selection for Gaussian backend `MeasureFock` if `shots=1`? @nquesada

Reply:
I moved this part outside of `if shots != 1:` since the `select` keyword doesn't actually seem to matter (and post-selection didn't seem to be supported for single-shots with the Gaussian backend).